SpatialGDK/ExternalSchemaCodegen/Programs/Improbable.CodeGen.Unreal/HeaderGenerator.cs

```diff
@@ -0,0 +1,376 @@
+using Improbable.CodeGen.Base;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Improbable.CodeGen.Unreal
+{
+    public static class HeaderGenerator
+    {
+        private static string DefaultHeaderIncludes = string.Join(Environment.NewLine, new []{
+            "#include <cstddef>",
+            "#include <cstddef>",
+            "#include <cstdint>",
+            "#include <functional>",
+            "#include <set>",
+            "#include <string>",
```

Comment:
Why do we need std::string?

Reply:
accidental legacy from cpp codegen, will remove